frontend/assets/javascripts/src/main.js

```diff
@@ -26,7 +26,8 @@ require([
     'src/modules/patterns',
     'src/modules/memstatus',
     'src/modules/faq',
-    'src/modules/landingBundles'
+    'src/modules/landingBundles',
+    'src/modules/bundlesLanding'
```

Comment:
Uh-oh. Which is what? I mean, what does which? Maybe refactor this soon after release?